frontend/amundsen_application/static/js/pages/TableDetailPage/constants.ts

```diff
@@ -7,7 +7,7 @@ export const EDIT_OWNERS_TEXT = 'Click to edit owners in';
 
 export const COLUMN_URL_KEY = 'column';
 export const DATE_RANGE_TITLE = 'Date Range';
-export const DESCRIPTION_TITLE = 'Description';
+export const DESCRIPTION_TITLE = 'Editable Description';
```

Comment:
rename the default description to editable description

Reply:
@wli-brex any suggestion on how to name the "editable description"? maybe "comment section"?

Reply:
discussed during the meeting. Will update this to "comment"